src/components/Theme/style.css

```diff
@@ -138,6 +136,12 @@
   }
 }
 
+.actions .btn-primary {
```

Comment:
will this work? `.actions` is for the Actions component

Reply:
if this is style is truly just for the Confirm Action button, then it's right to be under the `Confirm/style.css`. The theme style should not have any concept of other components.

Reply:
The `actions` class is in the Theme stylesheet to handle cases in which we have more than one button. At the moment it only happens in one case (Confirm screen) but I don't see why we wouldn't reuse this container elsewhere.

Reply:
I've created a separate issue for that
https://jira.onfido.co.uk/browse/CX-1541